smac/optimizer/smbo.py

```diff
@@ -365,7 +367,7 @@ def validate(self, config_mode='inc', instance_mode='train+test',
                                         output_fn=new_rh_path)
         return new_rh
 
-    def _get_timebound_for_intensification(self, time_spent:float):
+    def _get_timebound_for_intensification(self, time_spent: float = 0):
```

Comment:
Could you please remove this default value? The user should always have to provide it to avoid mistakes.